### PR TITLE
E2e/join support consecutive ip

### PR DESCRIPTION
**Pull request by hellomyboy**
### Describe what this PR does / why we need it
join sub command support consecutive IP  range, example : sealer join -m xxx.xxx.xxx.1-xxx.xxx.xxx.5 -n xxx.xxx.xxx.11-xxx.xxx.xxx.15

### CODE REVIEW

**Comment by fanux** [comment]
delete the .DS_Store file

File: apply/run.go [hunk 1]
```diff
@@ -61,6 +63,47 @@ func IsIPList(args string) bool {
...
+	}
+	return big.NewInt(0).SetBytes(ip.To16())
+}
+
+func CompareIP(v1, v2 string) int {
+	i := ipToInt(v1)
+	j := ipToInt(v2)
+
+	if i == nil || j == nil {
+		os.Exit(-1)
```
Comment (fanux) [inline apply/run.go:84]: should not Exit, return error instead.

File: apply/run.go [hunk 2]
```diff
@@ -61,6 +63,47 @@ func IsIPList(args string) bool {
 	return true
 }
 
+func PreProcessIPList(joinArgs *common.RunArgs) {
```
Comment (fanux) [inline apply/run.go:66]: Provide some test cases for this function.
Attention about the edge cases like: `x.x.x.x-` x.x.x.x-x.x.x.x-x.x.x.x` ...
